lib/dartdoc.dart

```diff
@@ -107,18 +107,31 @@ class Dartdoc extends PackageBuilder {
     outputDir = Directory(config.output)..createSync(recursive: true);
   }
 
-  /// An asynchronous factory method that builds Dartdoc's file writers
-  /// and returns a Dartdoc object with them.
-  static Future<Dartdoc> withDefaultGenerators(
```

Comment:
I suggest leaving this in place with @deprecated and we'll remove it with the next breaking change. withDefaultGenerators is one of the few things I know people externally use.. https://github.com/search?q=Dartdoc.withDefaultGenerators&type=Code

Reply:
Done